runelite-client/src/main/java/net/runelite/client/plugins/menuentryswapper/MenuEntrySwapperConfig.java

```diff
@@ -227,14 +227,14 @@ default boolean swapBank()
 	}
 
 	@ConfigItem(
-		keyName = "swapBirdhouseEmpty",
+		keyName = "swapBirdhouse",
```

Comment:
Since the key name is being changed, we should migrate config values. (eg. `true` -> `EMPTY`, and `false` -> `Interact`)